homeassistant/components/update/__init__.py

```diff
@@ -138,7 +138,7 @@ async def async_install(entity: UpdateEntity, service_call: ServiceCall) -> None
             f"Update installation already in progress for {entity.name}"
         )
 
-    await entity.async_install_with_progress(version, backup)
+    await entity.async_install_with_progress(version, backup or False)
```

Comment:
Use `service_call.data[ATTR_BACKUP]` on line 131 since it now has a default, and you don't need `or False` here

Reply:
Done in [`921ebaa` (#68553)](https://github.com/home-assistant/core/pull/68553/commits/921ebaaff7d0c8e53e34f26e33cb5fc1e1e5949b)